types/use-sync-external-store/use-sync-external-store-tests.ts

```diff
@@ -1,5 +1,5 @@
-import { useSyncExternalStore } from 'use-sync-external-store';
-import { useSyncExternalStoreExtra } from 'use-sync-external-store/extra';
+import { useSyncExternalStore } from 'use-sync-external-store/shim';
+import { useSyncExternalStoreWithSelector } from 'use-sync-external-store/shim/withSelector';
```

Comment:
should be`shim/with-selector`. sorry, I didn't notice either when reviewed.

Reply:
Whoops, thanks for the report! Fixed in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/56922